### ISSUE TITLE
[OTHER] Reorganize media assets.

### ISSUE THREAD

**Issue by sameer1612** [thecyberworld/thecyberhub.org#231]
### What would you like to share?

- [ ] Compress and re-upload images/videos to cloud storage.
- [ ] Update links in the codebase.
- [ ] Remove media from assets.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by sameer1612** [comment]
@kabir0x23 you can optimize media files using 
https://compresspng.com
https://www.veed.io/tools/video-compressor

**Comment by kabir0x23** [comment]
> @kabir0x23 you can optimize media files using https://compresspng.com https://www.veed.io/tools/video-compressor

Thanks for the links and issue: i am on it 💪🏽

**Comment by kabir0x23** [comment]
Hey @sameer1612 
Have a Bad NEWS

quoto exceded
![image](https://user-images.githubusercontent.com/44284877/197123340-566ec592-2332-4b96-88dc-a539afddffc5.png)

**Comment by kabir0x23** [comment]
exmaple github:

if i share an image or video here, it will be uploaded in the github server 
then we can use these links in our website 

example:
https://user-images.githubusercontent.com/44284877/197123340-566ec592-2332-4b96-88dc-a539afddffc5.png

**Comment by sameer1612** [comment]
I wonder if you actually got **1.7K** hits in 2 days. Anyhow quota is set to 1gb per day, it will be reset tonight. High bandwidth is also due to the large uncompressed files you have right now. once compressed this usage will reduce drastically.

Putting files on GitHub may not be a good idea, if the request rate is high then fair chance that they'll blacklist your domain. Will suggest compressing and keeping image assets locally then. Use storage for dynamic uploads like maybe the user's profile photo, certificates, etc.

**Comment by kabir0x23** [comment]
> I wonder if you actually got 1.7K hits in 2 days.

Yeah, that's the community website and we are providing everything for free, that;s the one of the main reason:
![image](https://user-images.githubusercontent.com/44284877/197125179-e48b5a5b-8b59-43f3-80cd-73269a79c3a9.png)


> Putting files on GitHub may not be a good idea, if the request rate is high then fair chance that they'll blacklist your domain

Yes this would be, didn't cam in my mind, thanks

> Will suggest compressing and keeping image assets locally then. Use storage for dynamic uploads like maybe the user's profile photo, certificates, etc.

This is the better option and will try my best to build a full stack soon and transfer it to aws/azure

**Comment by sameer1612** [comment]
@kabir0x23 lets compress them the best and then we'll look into where to keep. Keep them in repo itself, no issues. Smaller files will give better UX. 


_One out of context update: I have setup next and apis. We can connect this weekend and wrap that work. _

**Comment by kabir0x23** [comment]
> lets compress them the best and then we'll look into where to keep. Keep them in repo itself, no issues. Smaller files will give better UX.
 
okay ✔ Compressing right now.

> _One out of context update: I have setup next and apis. We can connect this weekend and wrap that work. _

Awesome 😄 and thanks for your efforts.
